GNUmakefile

```diff
@@ -136,7 +136,9 @@ ifndef CFLAGS
     endif
 endif
 
-CFLAGS += -Wall -Werror -Wshadow -Wextra -Iinc \
+CFLAGS += -Wall -Werror -Wshadow -Wextra \
+		  -Wno-unused-parameter \
```

Comment:
Sorry can you explain (or explain again, if you've already explained it to me and I forgot) why we want -Wno-unused-parameter?  The code was as far as I know clean with respect to this warning, so why do we need to disable the unused-parameter warning now?